rotkehlchen/chain/evm/decoding/aave/v2/decoder.py

```diff
@@ -0,0 +1,82 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.ethereum.utils import asset_normalized_value
+from rotkehlchen.chain.evm.decoding.aave.common import Commonv2v3Decoder
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.utils.misc import hex_or_bytes_to_address, hex_or_bytes_to_int
+
+from ..constants import AAVE_LABEL, CPT_AAVE_V2
+from .constants import BORROW, DEPOSIT, REPAY
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.chain.evm.decoding.structures import DecoderContext
+    from rotkehlchen.chain.evm.node_inquirer import EvmNodeInquirer
+    from rotkehlchen.types import ChecksumEvmAddress
+    from rotkehlchen.user_messages import MessagesAggregator
+
+
+class Aavev2EvmDecoder(Commonv2v3Decoder):
```

Comment:
For this kind of decoers we use `common` in the name like `GitcoinV2CommonDecoder`, `MoneriumCommonDecoder`, `ZeroxCommonDecoder`, `OneinchCommonDecoder` (there are some that not but I believe those are the exception for special reasons). Change the name from `Aavev2CommonDecoder` and likewise for v3